src/service/main.py

```diff
@@ -0,0 +1,35 @@
+BASE_URL = "http://localhost:15672/api/"
```

Comment:
Please remove this script from here since it is not a service. Put everything into test_integration_ scripts.